draft-xyz-rats-psa-endorsements.md

```diff
@@ -131,94 +141,173 @@ firmware measurement associated with Implementation ID
-firmware RoT descriptors that have PATCH number 0 and "adjacent" MINOR numbers
-as illustrated in {{fig-updates-patches}}.  Note that changing the MAJOR
-version number would typically result in a separate product / upgrade chain.
+Identity Claims carry the verification key associated with the Initial
+Attestation Key (IAK) of a PSA device.
+
+Each verification key is encoded alongside the corresponding device Instance ID
+in a CoMID identity claim.
+
+The Instance ID is encoded using `tagged-ueid-type` as `$device-id-type-choice`.
+
+The IAK public key is encoded as a COSE Key according to Section 7 of
+{{!RFC8152}} and wrapped in the `COSE_KeySet`. The number of items in the
+`COSE_KeySet` MUST be 1.
+
+The example in {{ex-identity-claim}} shows a CoMID identity claim carrying a
+secp256r1 EC public IAK associated with Instance ID `4ca3...d296`.
+
+An Identity CoMID can carry as many Identity Claims as needed.
 
-~~~ goat
-{::include art/updates-patches.txt}
 ~~~
-{: #fig-updates-patches title="Updates, Patches Relations and Semantic Versioning"}
+{::include examples/instance-pub.diag}
+~~~
+{: #ex-identity-claim title="Example Identity Claim"}
 
-The Reference Value CoMID of the patching or updating firmware has a
-`linked-tag-entry` map populated as follows:
 
-* key 0 contains the tag identifier of the Reference Value CoMID of the patched
-  or updated firmware;
-* key 1 contains one of `comid-patches` or `comid-updates` relations.
+## Certification
+{: #sec-certificates}
 
-{{ex-linked-tag}} provides an example of a Reference Value CoMID patching
-another Reference Value CoMID with a UUID tag identifier
-`3f06af63-a93c-11e4-9797-00505690773f`.
+A PSA Certification can certify PSA hardware, software RoT or the complete device.
+Specifically, it can certify one of the following:
+
+* The underlying platform, identified by the Impementation ID
+
+* One or more firmware components of the mutable PSA RoT.
+
+* To entire PSA device.
+
+PSA Certification claim contains the metadata about the certificate. The claim is 
+expressed by creating a new CoMID Tag for provisioning certification details.
+
+The certification status is encoded as a CoMID endorsement as follows:
+
+* The Implementation ID of the PSA RoT to which the certification metadata apply, is encoded 
+  using the class-id-type-choice (2) entry in the `element-name` map with a `tagged-impl-id` type;
+
+* The metadata associated with the certification is encoded in a `comid.psa-cert-meta` structure
+  which extends the `endorsed-value-map`.
+  
+* Optionally, the CoMID linked tags can contain one or more entries of linked-tag-map, each entry carries
+  tag id of CoMID, which it certifies and tag-rel type set to `comid.supplements` relation,
+  as illustrated in {{fig-cert-link}}.
+
+A PSA Certification data element has the following attributes:
+
+* `cert-level-type-choice`    : PSA certification level.  Acceptable values are x, y, z...
```

Comment:
x, y and z are not acceptable values :-)

Reply:
I will remove this comment as CDDL is now clear to state how exactly it is set..

No need to explain here!